pkg/network/ebpf/c/protocols/http2/decoding-tls.h

```diff
@@ -517,10 +533,27 @@ int uprobe__http2_tls_handle_first_frame(struct pt_regs *ctx) {
         iteration_value->frames_count = 1;
     }
     dispatcher_args_copy.off += current_frame.length;
+    // We're exceeding the packet boundaries, so we have a remainder.
+    if (dispatcher_args_copy.off > dispatcher_args_copy.len) {
```

Comment:
what about dispatcher_args_copy.off = dispatcher_args_copy.len? isn't it possible to happen?

Reply:
If `dispatcher_args_copy.off == dispatcher_args_copy.len`, that means the frame perfectly fits in the buffer, in which case we don't need to store a remainder.

Reply:
You are right. thanks